NitroxServer/GameLogic/Entities/EntitySimulator.cs

```diff
@@ -92,11 +91,10 @@ private List<Entity> AssignForCells(Player player, AbsoluteEntityCell[] added)
             foreach (AbsoluteEntityCell cell in added)
             {
                 List<Entity> entities = entityManager.GetEntities(cell);
-
                 assignedEntities.AddRange(
                     entities.Where(entity => cell.Level <= entity.Level &&
-                                                ((entity.SpawnedByServer && serverSpawnedSimulationWhiteList.Contains(entity.TechType)) || !entity.SpawnedByServer) &&
-                                                simulationOwnershipData.TryToAcquire(entity.Id, player, DEFAULT_ENTITY_SIMULATION_LOCKTYPE)));                       
+                                             (entity.SpawnedByServer && serverSpawnedSimulationWhiteList.Contains(entity.TechType) || !entity.SpawnedByServer) &&
```

Comment:
I think we should leave this as-is to not have any ambiguity.

Reply:
I prefer not to have redundant parenthesis in the code because the auto formatting will clean it up and changing it to never clean it up also seems undesirable.

How about this?

```cs
bool shouldBeOwnershipSimulated = entity.SpawnedByServer && serverSpawnedSimulationWhiteList.Contains(entity.TechType) || !entity.SpawnedByServer;
bool canAquireLockForPlayer = simulationOwnershipData.TryToAcquire(entity.Id, player, DEFAULT_ENTITY_SIMULATION_LOCKTYPE);
return cell.Level <= entity.Level && shouldBeOwnershipSimulated && canAquireLockForPlayer;
```